osbs/utils.py

```diff
@@ -316,12 +318,14 @@ def make_name_from_git(repo, branch, limit=53, separator='-'):
 
     repo = git_repo_humanish_part_from_uri(repo)
     branch = branch or 'unknown'
+    hash_str = b64encode(sha256(repo + branch).hexdigest())[:hash_size]
```

Comment:
It's about packing those 5 characters with useful information. The entropy of the first five characters of the hexdigest is much lower than that of the first five characters of the base64 encoded hexdigest (it uses upper and lower case, and more than the first 6 characters of the alphabet, etc).